recipes/crbeam/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "crbeam" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/okolo/mcray/archive/refs/tags/v1.0.6rc1.tar.gz
+    sha256: 0ef92a6116394dd7926f0a7ee446a6d9ef7c79c904b63d6689fe49751ffae035
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - gxx_linux-64 ==11.2
```

Comment:
Please use the compiler macros

Reply:
@xhochy thanks for suggestion. I tried to use compiler('cxx') and compiler('fortran') before, but I encountered linker issues presumably related to OpenMP.

Reply:
Please use them and then let's look together at the errors. We will not merge with explicitly listed compilers here.